indico/web/templates/forms/location_widget.html

```diff
@@ -3,7 +3,7 @@
 {% block html %}
     <div class="i-location-field" id="{{ field.id }}-wrapper">
         <input type="hidden" id="{{ field.id }}" name="{{ field.name }}"
-               value="{{ field._value() | tojson | forceescape }}">
+               value="{{ field._value() | tojson | forceescape }}" autofocus="true">
```

Comment:
boolean attributes in HTML are set by having them; their value doesn't matter. so you just want just `autofocus` (since e.g. `autofocus=false` would still enable autofocus since the value is ignored)